.readthedocs.yml

```diff
@@ -1,8 +1,8 @@
 build:
   image: latest
-  
+
 python:
-  version: 3.8
+  version: 3.7
```

Comment:
Python 3.7 has basically reached its end of life: https://endoflife.date/python. I think 3.9 or 3.10 would be better here (depending a bit on what RTD offers)

Reply:
Agree, we can increase the version to 3.10 or 3.9 in a next pr.